core/src/sigverifier/bls_sigverifier.rs

```diff
@@ -228,19 +229,18 @@ mod tests {
         bitmap.set(3, true);
         bitmap.set(5, true);
         let vote_rank: usize = 2;
+
+        let certificate =
+            CertificateId::from_certificate_type(CertificateType::Finalize, Some(4), None);
```

Comment:
Hmm, I'm not sure about this. `CertificateId` was created to uniquely identify a `Certificate`. And `from_certificate_type` makes me expect that you are only giving `CertificateType` as input, in reality you are giving the hashes as well. You are only defining `Certificate::new` here right?